src/components/projectSelect/index.tsx

```diff
@@ -13,6 +13,12 @@ import {
 import { GET_PROJECTS, GET_VIEWABLE_PROJECTS } from "gql/queries";
 import { ProjectOptionGroup } from "./ProjectOptionGroup";
 
+type project = {
```

Comment:
[nit] types are capitalized
```suggestion
type Project = {
```